.pre-commit-config.yaml

```diff
@@ -13,8 +13,6 @@ repos:
   hooks:
     - id: forbid-crlf
     - id: remove-crlf
-    - id: chmod
-      args: ['644']
```

Comment:
You can add a `exclude: .pre-commit-config.yaml` line to exclude 644 permissions on this file if I'm not mistaken. See https://pre-commit.com/#config-exclude

Reply:
Oh wait sorry, I misread things. Let me see if there's another way...

Reply:
Since we only care about if files have execute permission, so I thought `chmod -x` should work. Unfortunately the hook doesn't support it yet https://github.com/Lucas-C/pre-commit-hooks/blob/a30f0d816e5062a67d87c8de753cfe499672b959/pre_commit_hooks/chmod.py#L35-L40.

Reply:
Maybe we skip the `chmod` hook on pre-commit.ci using https://pre-commit.ci/#configuration-skip? Since we have the check on the `Style Checks' CI already.